prime-router/src/test/kotlin/azure/batch/UniversalBatchFunctionTests.kt

```diff
@@ -448,10 +455,10 @@ class UniversalBatchFunctionTests {
         every { mockTask.schemaName } returns schemaName
         every { mockTask.bodyFormat } returns bodyFormat
         every { engine.generateEmptyReport(any(), any()) } returns Unit
-        every { engine.db.fetchAndLockBatchTasksForOneReceiver(any(), any(), any(), any(), any()) } returns listOf(
+        every { mockDb.fetchAndLockBatchTasksForOneReceiver(any(), any(), any(), any(), any()) } returns listOf(
```

Comment:
If you had to make this change on this line then that means you are not properly passing mockDb to the engine object? You are passing mockDb to engine on 437, so why doesn't engine.db work here? Even though this test works, I think it's messy and can get cleaned up because it's not clear what function/purpose the object on line 54 has?